test/e2e/ui/dashboard.go

```diff
@@ -54,29 +55,22 @@ var _ = SIGDescribe("Kubernetes Dashboard", func() {
 
 		By("Checking to make sure we get a response from the kubernetes-dashboard.")
 		err = wait.Poll(framework.Poll, serverStartTimeout, func() (bool, error) {
-			var status int
-			proxyRequest, errProxy := framework.GetServicesProxyRequest(f.ClientSet, f.ClientSet.CoreV1().RESTClient().Get())
-			if errProxy != nil {
-				framework.Logf("Get services proxy request failed: %v", errProxy)
-			}
-
 			ctx, cancel := context.WithTimeout(context.Background(), framework.SingleCallTimeout)
 			defer cancel()
 
-			// Query against the proxy URL for the kube-ui service.
-			err := proxyRequest.Namespace(uiNamespace).
-				Context(ctx).
-				Name(uiServiceName).
+			var status int
+			err = f.ClientSet.CoreV1().RESTClient().Get().
+				AbsPath(uiRedirect).
```

Comment:
don't add new tests for `/ui` (https://github.com/kubernetes/kubernetes/pull/53766 is queued to remove that redirect), switch to calling the correct service proxy endpoint.

Reply:
also, the point of this first test was to check the direct service proxy. the change here makes this a duplicate of the test online 82

Reply:
@liggitt I can remove this test as a duplicate. I could not be kept as it was because it did not work with HTTPS. Would it be enough for now?

Reply:
> I can remove this test as a duplicate. I could not be kept as it was because it did not work with HTTPS. 

We need to update this test to work with https by adjusting the `Name` segment of the request, not switch it to checking the /ui redirect:

```
Name(utilnet.JoinSchemeNamePort("https", uiServiceName, ""))
```

Reply:
Done. Thanks.